app.go

```diff
@@ -26,6 +26,7 @@ import (
 	"time"
 
 	"github.com/fxamacker/cbor/v2"
+	"github.com/gofiber/fiber/v3/binder"
```

Comment:
I'm confused as to why we are doing msgpack via binder instead of how cbor is being done?

Reply:
i think so, but lemme do it in another PR.

Reply:
Sounds good to me

Reply:
@hotrungnhan https://github.com/gofiber/fiber/issues/3577